docs/spec/blockchain/blockchain.md

```diff
@@ -56,8 +56,8 @@ type Header struct {
     ConsensusParamsHash []byte  // SimpleMerkle of the ConsensusParams
 
     // consensus
-    Proposer            []byte  // Address of the block proposer
     EvidenceHash        []byte  // SimpleMerkle of []Evidence
+    ProposerAddress     []byte  // Address of the original proposer of the block
```

Comment:
It's a bit strange to call it just `Proposer` because it's really an address (without pubkey or power).